packages/web/src/common/store/pages/profile/selectors.ts

```diff
@@ -39,6 +40,37 @@ export const getProfileFeedLineup = (state: CommonState) =>
 export const getProfileTracksLineup = (state: CommonState) =>
   state.pages.profile.tracks
 
+export const getProfileCollections = createDeepEqualSelector(
```

Comment:
The selectors in this file scare me a bit. There must be a better way. This is certainly good though, and not a new pattern.

Reply:
yeah this file is wild! essentially I just stripped out the relevant parts for collections from `getProfileUser`